sklearn/svm/_classes.py

```diff
@@ -461,6 +473,7 @@ class SVC(BaseSVC):
 
     shrinking : bool, default=True
         Whether to use the shrinking heuristic.
+        See the :ref:`User Guide <shrinking>`.
```

Comment:
it'll probably be safer to call it `svm_shrinking` or something (to avoid potential duplicate keys)